spec/features/admin/admin_users_spec.rb

```diff
@@ -80,8 +82,8 @@
     end
 
     it "should have user edit page" do
-      page.should have_content("Name")
-      page.should have_content("Password")
+      expect(page).to have_content("Name")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.